### ISSUE TITLE
Investigate SLO options

### ISSUE THREAD

**Issue by lfrancke** [stackabletech/issues#189]
e.g. https://keptn.sh/

Can we somehow specify SLO/SLAs and then make sure that maintenance windows etc. always fit wtihin those windows and/or at lesat track the current SLIs (indicators) in each object

**Comment by NickLarsenNZ** [comment]
You could have it so that maintenance could be done if there is enough error budget left (still with some margin to allow for unexpected outages), or deferred until the next period (month/quarter).

**Comment by NickLarsenNZ** [comment]
I don't think we should specify SLOs, only SLIs. The user (being the people operating the product) would then specify their SLOs based on the available SLIs.
